src/app/core/campaign/campaign-store.service.ts

```diff
@@ -227,11 +227,12 @@ export class CampaignStoreService {
     startAt = new Date(startAt.valueOf()).toISOString().slice(0, 10);
     endAt = new Date(endAt.valueOf()).toISOString().slice(0, 10);
     const loading = this._campaign$.pipe(
-      mergeMap(state => {
```

Comment:
I was seeing a cancelled extra request in here, so I moved to concatMap to allow the request to finish before starting a new one, and that fixed the extra cancelled request.